gear-programs/erc20-relay/app/src/services/erc20_relay.rs

```diff
@@ -0,0 +1,274 @@
+// Incorporate code generated based on the IDL file
+#[allow(dead_code)]
+mod vft {
+    include!(concat!(env!("OUT_DIR"), "/vft-gateway.rs"));
+}
+
+use super::*;
```

Comment:
Please don't use `*` imports enywhere. It complicates code-review and navigation in source files. For example, if I want to see where the `State` is declared I need to go to `super` crate (and if there're some re-exports in `super` I also need to search among all of them)

Reply:
However I wonder why it makes issues. It is common pattern, for example `use sails_rs::{cell::RefCell, prelude::*};` - https://github.com/gear-tech/sails/blob/master/examples/demo/app/src/lib.rs#L4

Anyway I will edit it if you insist ).